scripts/content_type_stats.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*
+
+"""
+Collects content-type data from the corpus.
+Operates with gz files.

```

Comment:
This information is not really important, and it might change in the future (we are using `openall` anyway), so let's just skip it. Rather, elaborate a bit on what we actually collect.